src/common/Constants.ts

```diff
@@ -2,6 +2,14 @@
 // in the HubPool's proposeRootBundle method should be: Mainnet, Optimism, Polygon, Boba, Arbitrum
 export const CHAIN_ID_LIST_INDICES = [1, 10, 137, 288, 42161];
 
+export const MAX_RELAYER_DEPOSIT_LOOK_BACK: { [chainId: number]: number } = {
+  1: 11500,
+  10: 350000,
+  137: 70000,
+  288: 6000,
+  42161: 35000,
+};
+
```

Comment:
This is missing a zero right? Sanity check is that block lookbacks for Arbitrum and Optimism should roughly be the same since they both process ~2 blocks/second
```suggestion
  42161: 350000,
};

```

Reply:
Well spotted - thanks.